docs/technical-reference/controllers.md

```diff
@@ -31,6 +31,13 @@ These are not only able to synchronize the Kyma resource in the remote,
 but they also replace the specification for all further parts of the reconciliation as a _virtual_ Kyma.
 For more information, checkout the `ReplaceWithVirtualKyma` function.
 
+## Mandatory Modules Controller
+
+[Mandatory Modules Controller](../../internal/controller/mandatory_modules_controller.go) deals with the reconciliation of mandatory modules.
+
+The Channel concept does not apply to Mandatory Modules, so the controller fetches all the Mandatory ModuleTemplate CRs without any channel filtering.It then translates the ModuleTemplate CR for mandatory Module to [Manifest CR](/api/v1beta2/manifest_types.go) with an OwnerReference to the Kyma CR. Similar to the Kyma Controller, 
+it propagates changes from the ModuleTemplate CR to the Manifest CR. Mandatory ModuleTemplate CR is not synchronized to the remote cluster and the Module Status does not appear in the Kyma CR Status.
```

Comment:
```suggestion
it propagates changes from the ModuleTemplate CR to the Manifest CR. Mandatory ModuleTemplate CR is not synchronized with the remote cluster and the module status does not appear in the Kyma CR status.
```
Is that any reason to capitalize "module" and "status"?

Reply:
Not really. For some reason, I thought we capitalized them in the rest of the docs, but it seems I am mistaken